root/static/scripts/edit/URLCleanup.js

```diff
@@ -1223,6 +1223,28 @@ const CLEANUPS = {
       return url;
     }
   },
+  niconicouser: {
```

Comment:
`niconicouser` can probably be merged into `niconicovideo` on the model of `dailymotion`.

Reply:
Should I merge the clean function with this? If they are merged
`(/^(?:https?:\/\/)?nicovideo\.jp\/(?:user|watch)\/(?[0-9]+|sm[0-9]+).*$/`

Reply:
Don't merge it with `dailymotion` (if `dailymotion` is _this_ is your question).

Reply:
no, it's not dailymotion.
It's niconicouser and niconicovideo

Reply:
Oh, ok! There are three issues: subdomain is not matched anymore, `user` and numeric id `[0-9]+` are not tied together, and there is two capturing groups instead of one. All fixed with:

`/^(?:https?:\/\/)?(?:[^\/]+\.)?nicovideo\.jp\/(user\/[0-9]+|watch\/sm[0-9]+).*$/`